src/test/java/com/VyTrack_Project/step_definitions/CalendarEvenStepDefinitions.java

```diff
@@ -0,0 +1,68 @@
+package com.VyTrack_Project.step_definitions;
+import com.VyTrack_Project.pages.CalendarEventsPage;
+import com.VyTrack_Project.pages.LoginPage;
+import com.VyTrack_Project.utilities.BrowserUtils;
+import com.VyTrack_Project.utilities.ConfigurationReader;
+import com.VyTrack_Project.utilities.Driver;
+import io.cucumber.java.en.*;
+import io.cucumber.java.zh_tw.並且;
+import net.bytebuddy.asm.Advice;
+import org.checkerframework.checker.units.qual.C;
+import org.junit.Assert;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+
+import java.util.Map;
+
+public class CalendarEvenStepDefinitions{
+
+    CalendarEventsPage calendarEventsPage = new CalendarEventsPage();
+
+    LoginPage loginPage = new LoginPage();
+    @Given("the user logged in with blow correct credentials")
+    public void the_user_logged_in_with_blow_correct_credentials(Map<String,String> credentials) {
+        loginPage.userName.sendKeys(credentials.get("username") );
+        loginPage.password.sendKeys(credentials.get("password"));
+        loginPage.submit.click();
+    }
```

Comment:
Hi Gulzina! You did worked on the login function from scratch:) It's working and all good but keep in mind next time there are ready solutions for it already implemented to framework.